examples/SDL2/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include<iostream>
+#include<SDL2/SDL.h>
+
+int main(int argc, char *argv){
```

Comment:
Plus some of the windows failures are because it cant find dxguid.lib, directX related